<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doc_edit</class>
 <widget class="QWidget" name="doc_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入你的姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="genderEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入你的性别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="roomEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入你的科室</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contactEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入你的联系方式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="majorEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入你的专业领域</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入你的工作年限</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="positionEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入你的职位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>录入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>doc_edit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
